build/print.go

```diff
@@ -244,8 +244,8 @@ const (
 	precCmp
 	precAdd
 	precMultiply
-	precSuffix
 	precUnary
+	precSuffix
```

Comment:
revert?

Reply:
I did it on purpose, otherwise `-f(x)` gets formatted as `-(f(x))`. That wasn't the case previously because it used to be parsed as `{-f}{(x)}`, and is parsed now as `-{f(x)}`, which is more correct.

Reply:
Ah sorry, I misread. Looks good.